Source/DafnyCore/Compilers/Compiler-go.cs

```diff
@@ -2061,12 +2061,36 @@ private class SpecialNativeType : UserDefinedType {
         wr.Write((bool)e.Value ? "true" : "false");
       } else if (e is CharLiteralExpr) {
         var v = (string)e.Value;
-        wr.Write("_dafny.Char('{0}')", TranslateEscapes(v, isChar: true));
```

Comment:
Can we keep the complexity in a separate method (as it was with TranslateEscapes)?  Maybe `TranslateCharLiteral`

Reply:
Yeah I like that, and even better to use `TrStringLiteral` to encapsulate the even bigger string case/